integration_tests/tests/integration_tests/token_accounts_tests.rs

```diff
@@ -29,7 +29,7 @@ async fn test_get_token_accounts_by_mint() {
 
     let request = r#"
     {
-        "mintAddress":"wKocBVvHQoVaiwWoCs9JYSVye4YZRrv5Cucf7fDqnz1"
```

Comment:
Similar to owner with accepting mint and mintAddress. Marking mintAddress as deprecated.